tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -119,3 +126,53 @@ Verify Quick Starts Work As Expected When At Least One Step Is Skipped
     END
     Run Keyword And Continue On Failure     QuickStart Status Should Be    ${element}      In Progress
     Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}  exp_link_text=Continue
+
+External URLs Should Not Be Broken
+    [Documentation]     Go through a QuickStart and checks the status of all the external links
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+    URLs HTTP Status Code Should Be Equal To    ${element_list}
+
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
+            Click Next OR TAB tab to click Sidewindow Button  ${sideWindowButton}
+            ${element_list}=    Get WebElements    xpath=//div[@Class="pf-c-drawer__panel-main"]//a[@href]
+            URLs HTTP Status Code Should Be Equal To    ${element_list}
+            ${Doc_Text}     Get Text  //*[@class="pf-c-drawer__body pf-m-no-padding pfext-quick-start-panel-content__body"]
+            ${Doc_links}     Get Regexp Matches   ${Doc_Text}   (?:(?:(?:ftp|http)[s]*:\/\/|www\.)[^\.]+\.[^ \n]+)
+            IF  ${Doc_links}
+                Log To Console      ${Doc_links}
+            END
+    END
+
+Verify Brokern Links in Quickstart
+    [Documentation]     Clicks on al the quick start and verify links
+    [Arguments]    ${quickStartElements}
+    ${quickStartCount}=   Get Length           ${quickStartElements}
+    ${TitleElements}=     Get WebElements      //div[@class="pf-c-card__title odh-card__doc-title"]
+    FOR    ${counter}    IN RANGE     ${quickStartCount}
+        Log To Console    "\n++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+        ${Title}=   Get Text          ${TitleElements[${counter}]}
+        ${Title}=   Split To Lines    ${Title}
+        Log To Console                ${Title[${0}]}
+        Click Element                 ${quickStartElements[${counter}]}
+        Wait Until Element Is Visible  //button[@class='pf-c-wizard__nav-link']
+        External URLs Should Not Be Broken
+    END
+
+Get QuickStart Items
+        Wait Until Resource Page Is Loaded
+        ${quickStartElements}=     Wait for QuickStart to Load
+        [Return]    ${quickStartElements}
+
+
+Click Next OR TAB tab to click Sidewindow Button
+    [Documentation]   Click next if next not found cick tab to find buttion
+    [Arguments]  ${sideWindowButton}
+    ${status}   Run Keyword And Return Status   Click Element  ${sideWindowButton}
+    IF  ${status} == False
+        Click Button    Next
+    END
```

Comment:
when does it happen to run these 3 lines? I mean, when the click element on `${sideWindowButton}` fail?

Reply:
yes correct

Reply:
no Berto that Tab thing will take care of it

Reply:
for the above, the  click is not failing always in case it is not able to  find the step button then it will click next after that TAB will take care of collecting all the link by looping  so we will o not miss any link